### PR TITLE
LRQA-44332 Add toggle to poshi execution to fail build on test failure

### PR DESCRIPTION
**Pull request by lesliewong92**
https://issues.liferay.com/browse/LRQA-44332

### CODE REVIEW

**Comment by lesliewong92** [comment]
ci:test:sf

**Comment by lesliewong92** [comment]
ci:test:relevant

File: build-test.xml [hunk 1]
```diff
@@ -9766,7 +9767,13 @@ if (ppstate.equals("normal")) {
 	</target>
 
 	<target name="run-poshi-runner">
-		<poshi-execute task="runPoshi" />
+		<condition property="failonerror" value="false">
+			<not>
+				<equals arg1="${failonerror}" arg2="true" />
+			</not>
+		</condition>
```
Comment (pyoo47) [inline build-test.xml]: @lesliewong92 Are you sure you need this condition task? It seems redundant when poshi-execute has a default value of false for the failonerror attribute.

Comment (lesliewong92) [reply build-test.xml]: If we don't have the condition, it'll set the attribute value to the String literal "${failonerror}", which will override the default attribute value.

Comment (lesliewong92) [reply build-test.xml]: As an alternative, I can change it to an <if> statement, and have branched <poshi-execute> statements depending on if _failonerror_ is set or not to make it more explicit.

**Comment by lesliewong92** [comment]
ci:test:sf

**Comment by lesliewong92** [comment]
ci:test:relevant

**Comment by pyoo47** [comment]
Pushed to BChan
https://github.com/brianchandotcom/liferay-portal/pull/64561

**Pull request closed without merging**